src/main/java/com/revlearn/team1/config/SecurityConfig.java

```diff
@@ -34,8 +34,15 @@ public UrlBasedCorsConfigurationSource corsConfigurationSource() {
 
     @Bean
     public CorsConfiguration corsConfiguration() {
+
+        // TODO add EC2 url to this list
```

Comment:
can you save this list at the top of the SecurityConfig class so its more accessible? i feel like this is useful info that might be used elsewhere in this file later down the line. Also keeps the corsConfiguration() function shorter and easier to read :)

Reply:
I feel like that makes things more difficult to read because a class variable indicates that the variable is used in many methods.  When it is included as a local variable in just the location that it will be used then its responsibility is clearer.

Reply:
sure okie!